.github/workflows/scripts/patch-changelog.sh

```diff
@@ -5,26 +5,24 @@
 set -euo pipefail
 
 check_not_empty() {
-    local VAR
-    typeset -n VAR
-    VAR="$1"
-    if [ -z "${VAR:-}" ]; then
-        echo "::error::Variable $1 is not set or empty"
-        exit 1
-    fi
+    for V in "$@"; do
+        typeset -n VAR="$V"
+        if [ -z "${VAR:-}" ]; then
+            echo "::error::Variable $V is not set or empty"
+            exit 1
+        fi
+    done
 }
```

Comment:
We have a duplication here - a sign to slowly think about the common library ;)

Reply:
Coming in the next PR!